### PR TITLE
Docs release 1.9.0 csi 3845 stretched

### PR DESCRIPTION
**Pull request by rivkap**

### CODE REVIEW

**Comment by samishekh** [comment]
hi it's sami

File: docs/content/release_notes/limitations.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The following limitations apply when using volume snapshots with the IBM block s
 -   When deleting a PersistentVolumeClaim (PVC), the persistent volume (PV) remains until all snapshots of the specific PV are deleted.
 -   When using the CSI (Container Storage Interface) driver with IBM Spectrum® Virtualize family products, a snapshot can only be used to provision a new volume of equal size.
 
-**Note:** For volume snapshot limitations pertaining specifically to HyperSwap usage, see [HyperSwap usage limitations](#hyperSwap-usage-limitations).
+**Note:** For volume snapshot limitations pertaining specifically to HyperSwap usage, see [High availability (HA) limitations](#high-availability-ha-limitations).
```
Comment (oriyarde) [inline docs/content/release_notes/limitations.md:16]: does IBM Docs support "nested" anchor links?
```suggestion
**Note:** For volume snapshot limitations pertaining specifically to HyperSwap usage, see [High availability (HA) limitations](#hyperswap-volumes).
```

File: docs/content/release_notes/limitations.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ The following limitations apply when using volume clones with the IBM block stor
 
 -   A PVC and its clone need to both have the same volume mode (**Filesystem** or **Block**).
 
-**Note:** For volume clone limitations pertaining specifically to HyperSwap usage, see [HyperSwap usage limitations](#hyperSwap-usage-limitations).
+**Note:** For volume clone limitations pertaining specifically to HyperSwap usage, see [High availability (HA) limitations](#high-availability-ha-limitations).
```
Comment (oriyarde) [inline docs/content/release_notes/limitations.md:28]: stretched has a clone limitation as well

File: docs/content/release_notes/limitations.md [hunk 3]
```diff
@@ -47,17 +47,22 @@ When a role switch is conducted, this is not reflected within the other orchestr
...
+### HyperSwap volumes
 The following IBM block storage CSI driver features are not supported on volumes where HyperSwap is used:
 
 - A HyperSwap volume cannot be created from a snapshot.
 
     **Note:** A snapshot can be created from a HyperSwap volume.
  - Volume cloning.
 
+### Stretched volumes
+ When conducting volume cloning both volumes must use stretched topology.
```
Comment (oriyarde) [inline docs/content/release_notes/limitations.md:64]:
```suggestion
 When conducting volume cloning, both volumes must use stretched topology.
```

File: docs/content/release_notes/limitations.md [hunk 4]
```diff
@@ -71,4 +76,4 @@ The following IBM block storage CSI driver features are not supported on volumes
  This limitation is tracked in the following places:
 
 - Red Hat Bug [2022328](https://bugzilla.redhat.com/show_bug.cgi?id=2022328)
-- Kubernetes Issue [106710](https://github.com/kubernetes/kubernetes/issues/106710)
\ No newline at end of file
+- Kubernetes Issue [106710](https://github.com/kubernetes/kubernetes/issues/106710)
```
Comment (oriyarde) [inline docs/content/release_notes/limitations.md]: what changed here?

File: docs/content/release_notes/whats_new.md [hunk 5]
```diff
@@ -4,6 +4,9 @@ IBM® block storage CSI driver 1.9.0 introduces the enhancements that are detail
 
 **General availability date:** 18 March 2022
 
+## Additional high availability (HA) feature
+
+Stretched volume and (FlashCopy) snapshot support is now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
```
Comment (oriyarde) [inline docs/content/release_notes/whats_new.md:9]: `support is now supported`?
```suggestion
Stretched volume and (FlashCopy) snapshot is now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
```

File: docs/content/release_notes/changelog_1.8.0.md [hunk 6]
```diff
@@ -1,10 +1,7 @@
 # 1.8.0 (December 2021)
 
 IBM® block storage CSI driver 1.8.0 added new support and enhancements:
-- New support and enhancements for IBM Spectrum Virtualize family storage systems:
-    - New HyperSwap support
-    - New NVMe®/FC support
-    - Increased StorageClass `SpaceEfficiency` parameter capabilities
+- New stretched topology support for SAN Volume Controller storage systems
```
[the remark this answers is not in the document]
Comment (rivkap) [reply docs/content/release_notes/changelog_1.8.0.md:4]: this was a mistake and should have been for 1.9.0. fixing

File: docs/content/configuration/creating_volumesnapshotclass_topology_aware.md [hunk 7]
```diff
@@ -41,4 +41,5 @@ After the YAML file is created, apply it by using the `kubectl apply -f` command
 kubectl apply -f <filename>.yaml
 ```
 
- The `volumesnapshotclass.snapshot.storage.k8s.io/<volumesnapshotclass-name> created` message is emitted.
\ No newline at end of file
+ The `volumesnapshotclass.snapshot.storage.k8s.io/<volumesnapshotclass-name> created` message is emitted.
+
```
Comment (oriyarde) [inline docs/content/configuration/creating_volumesnapshotclass_topology_aware.md:45]: do we want this extra new line?

File: docs/content/configuration/creating_volumestorageclass.md [hunk 8]
```diff
@@ -24,14 +24,16 @@ Use the `SpaceEfficiency` parameters for each storage system, as defined in [the
...
-- To create a volume with HyperSwap on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
+- To create a volume with high availability (HA) (HyperSwap or stretched topology) on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
   
   ```
   pool: demo-pool1:demo-pool2 
   ```
    **Important:** The two pools must be from different sites.
    
-  For more information about the HyperSwap limitations and requirements, see [Limitations](../release_notes/limitations.md) within the release notes and [Compatibility and requirements](../installation/install_compatibility_requirements.md) within this guide.
+  For more information about HyperSwap limitations, see [Limitations](../release_notes/limitations.md).
```
Comment (oriyarde) [inline docs/content/configuration/creating_volumestorageclass.md:34]: stretched has a limitation too

File: docs/content/release_notes/limitations.md (hunk 4 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md]: I think just spacing at the endof the file

Comment (oriyarde) [reply docs/content/release_notes/limitations.md]: can this be reverted? sounds like irrelevant noise for this PR

File: docs/content/configuration/creating_volumesnapshotclass_topology_aware.md (hunk 7 above)
Comment (oriyarde) [reply docs/content/configuration/creating_volumesnapshotclass_topology_aware.md:45]: I think this means there's one extra line in addition to what github likes. please try to remove 1

File: docs/content/release_notes/limitations.md (hunk 1 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md:16]: i'll try it out and update here the outcome

File: docs/content/release_notes/limitations.md (hunk 2 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md]: DONE

File: docs/content/release_notes/limitations.md (hunk 3 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md]: DONE

File: docs/content/release_notes/whats_new.md (hunk 5 above)
Comment (rivkap) [reply docs/content/release_notes/whats_new.md]: updated wording. DONE

File: docs/content/configuration/creating_volumestorageclass.md (hunk 8 above)
Comment (rivkap) [reply docs/content/configuration/creating_volumestorageclass.md]: updated. done

File: docs/content/release_notes/changelog_1.8.0.md [hunk 9]
```diff
@@ -5,6 +5,5 @@ IBM® block storage CSI driver 1.8.0 added new support and enhancements:
     - New HyperSwap support
     - New NVMe®/FC support
     - Increased StorageClass `SpaceEfficiency` parameter capabilities
-- Added custom resource configurability for the CSI driver health port
```
Comment (oriyarde) [inline docs/content/release_notes/changelog_1.8.0.md:8]: why was this removed?

File: docs/content/release_notes/whats_new.md [hunk 10]
```diff
@@ -4,6 +4,9 @@ IBM® block storage CSI driver 1.9.0 introduces the enhancements that are detail
 
 **General availability date:** 18 March 2022
 
+## Additional high availability (HA) feature
+
+Stretched volumes and (FlashCopy) snapshots are now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
```
Comment (oriyarde) [inline docs/content/release_notes/whats_new.md:9]: this might sound like snapshots are a new feature. how about:
```suggestion
Stretched volumes and stretched (FlashCopy) snapshots are now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
```

File: docs/content/release_notes/changelog_1.8.0.md (hunk 9 above)
Comment (rivkap) [reply docs/content/release_notes/changelog_1.8.0.md:8]: because it's in the 1.8.0 file and not 1.9

File: docs/content/release_notes/limitations.md (hunk 1 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md]: it seems to work. I've updated files accordingly. if there are places that I missed, let me know.

File: docs/content/release_notes/limitations.md (hunk 4 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md]: i'm actually not sure what this issue is. i think we should just resolve it. i think it's because we added in and changed placement of content for the stretched limitations

File: docs/content/release_notes/whats_new.md (hunk 10 above)
Comment (rivkap) [reply docs/content/release_notes/whats_new.md]: DONE

File: docs/content/configuration/creating_volumesnapshotclass_topology_aware.md (hunk 7 above)
Comment (rivkap) [reply docs/content/configuration/creating_volumesnapshotclass_topology_aware.md]: done

File: docs/content/release_notes/changelog_1.8.0.md (hunk 9 above)
Comment (oriyarde) [reply docs/content/release_notes/changelog_1.8.0.md:8]: I'm pretty sure the health port configurability was indeed [part of 1.8.0](https://github.com/IBM/ibm-block-csi-operator/blob/v1.8.0/config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml#L83), have anyone said otherwise?

File: docs/content/release_notes/limitations.md [hunk 11]
```diff
@@ -25,7 +25,7 @@ The following limitations apply when using volume clones with the IBM block stor
 
 -   A PVC and its clone need to both have the same volume mode (**Filesystem** or **Block**).
 
-**Note:** For volume clone limitations pertaining specifically to HyperSwap usage, see [HyperSwap usage limitations](#hyperSwap-usage-limitations).
+**Note:** For high availability volume clone limitations, see [High availability limitations](#high-availability-lilimtations).
```
Comment (oriyarde) [inline docs/content/release_notes/limitations.md:28]: please go over the diff here and verify that the new links actually work.
@matancarmeli7 could you please ask to prioritize CSI-3648 in the next sprint?
this type of check should be made by GitHub, not by reviewers

File: docs/content/release_notes/limitations.md (hunk 4 above)
Comment (oriyarde) [reply docs/content/release_notes/limitations.md]: I don't think it's related to the rest of the file.
could you try to remove 1 new line at the end of the file?
if github won't like it, then we'll keep this unrelated change

File: docs/content/release_notes/limitations.md (hunk 11 above)
Comment (matancarmeli7) [reply docs/content/release_notes/limitations.md:28]: done

File: docs/content/configuration/creating_volumesnapshotclass_topology_aware.md (hunk 7 above)
Comment (oriyarde) [reply docs/content/configuration/creating_volumesnapshotclass_topology_aware.md]: thanks. ok, now there's no apparent additional line, but there's still a change (same as in the [limitations file](https://github.com/IBM/ibm-block-csi-driver/pull/458#discussion_r812993257)).
could we try the same additional single line deletion here?

File: docs/content/release_notes/limitations.md (hunk 11 above)
Comment (oriyarde) [reply docs/content/release_notes/limitations.md:28]: thanks.
@rivkap let me know when you finish checking the links, so I can resolve this

Comment (rivkap) [reply docs/content/release_notes/limitations.md:28]: done and updated

File: docs/content/release_notes/limitations.md (hunk 4 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md]: done

File: docs/content/configuration/creating_volumesnapshotclass_topology_aware.md (hunk 7 above)
Comment (rivkap) [reply docs/content/configuration/creating_volumesnapshotclass_topology_aware.md]: deleted

File: docs/content/release_notes/changelog_1.8.0.md (hunk 9 above)
Comment (rivkap) [reply docs/content/release_notes/changelog_1.8.0.md]: fixed

File: docs/content/release_notes/changelog_1.8.0.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # 1.8.0 (December 2021)
 
-IBM® block storage CSI driver 1.8.0 added new support and enhancements:
+IBM® block storage CSI driver 1.8.0 adds new support and enhancements:
```
Comment (oriyarde) [inline docs/content/release_notes/changelog_1.8.0.md:3]: why this change?

File: docs/content/release_notes/limitations.md (hunk 11 above)
Comment (oriyarde) [reply docs/content/release_notes/limitations.md:28]: I don't see a change in this line

Comment (oriyarde) [reply docs/content/release_notes/limitations.md:28]: please verify your changes using:
image

File: docs/content/release_notes/changelog_1.8.0.md (hunk 12 above)
Comment (rivkap) [reply docs/content/release_notes/changelog_1.8.0.md:3]: because we do this between every release :)

File: docs/content/release_notes/limitations.md (hunk 11 above)
Comment (rivkap) [reply docs/content/release_notes/limitations.md:28]: this line specifically doesn't require a change. both hyperswap and stretched have these limitations

File: docs/content/release_notes/changelog_1.8.0.md (hunk 12 above)
Comment (oriyarde) [reply docs/content/release_notes/changelog_1.8.0.md:3]: then how come below 1.8.0 we still have `added`? what are the rules?

File: docs/content/release_notes/limitations.md (hunk 11 above)
Comment (oriyarde) [reply docs/content/release_notes/limitations.md:28]: @rivkap this conversation is not about linking to more precise locations, it's about not merging broken links to develop branch.
as I wrote in the first comment of this conversation, please ensure all links in this PR diff actually work.
e.g. the link in this line still doesn't work for me.
I try my best to cover all link changes when I check the diff, it would be extremely helpful if you always do the same **before** I do.
let me know if you need any clarifications about how to perform these checks.

Comment (rivkap) [reply docs/content/release_notes/limitations.md]: I checked all of the links on this PR and updated any that needed updating

File: docs/content/release_notes/changelog_1.8.0.md (hunk 12 above)
Comment (rivkap) [reply docs/content/release_notes/changelog_1.8.0.md:3]: we've been through this before :)
the new one is active / present.
the older content is past tense.

So in 1.8.0 it would have been "adds" and we would have changed 1.7.0 to "added" (for example) i do this for all RN content across all of the products.

Comment (oriyarde) [reply docs/content/release_notes/changelog_1.8.0.md:3]: I'm not sure I understand - in your 1.8.0 example, when (and why) do you change "added" to "adds" in relation to 1.8.0 GA?

File: docs/content/release_notes/limitations.md (hunk 11 above)
Comment (oriyarde) [reply docs/content/release_notes/limitations.md]: the link here still doesn't work for me.
let me know when you want me to show you how to check a link

Comment (rivkap) [reply docs/content/release_notes/limitations.md]: I had checked all of the links but must have missed this one.
I fixed it now and it should work 
image

File: docs/content/troubleshooting/log_status_collect.md [hunk 13]
```diff
@@ -53,7 +53,7 @@ Be sure to run the following steps and copy the output to an external file, when
 
     `PVC NAME: demo-pvc-file-system PVC STATUS: Bound`
 
-    **Note:** If the PVCs are not in the _Bound_ state collect the events of all unbound PVCs. (See [Log collection for unbound PVCs](#log-collection-for-unbound-pvcs).)
+    **Note:** If the PVCs are not in the _Bound_ state collect the events of all unbound PVCs. (See [Details collection for unbound PVCs](#Details-collection-for-unbound-pvcs).)
```
Comment (oriyarde) [inline docs/content/troubleshooting/log_status_collect.md:56]:
```suggestion
    **Note:** If the PVCs are not in the _Bound_ state, collect the events of all unbound PVCs. (See [Details collection for unbound PVCs](#Details-collection-for-unbound-pvcs).)
```

File: docs/content/release_notes/changelog_1.8.0.md (hunk 12 above)
Comment (oriyarde) [reply docs/content/release_notes/changelog_1.8.0.md]: thanks for reverting this change!

File: docs/content/troubleshooting/log_status_collect.md (hunk 13 above)
Comment (rivkap) [reply docs/content/troubleshooting/log_status_collect.md]: done

**Review by oriyarde — APPROVED** [review]
LGTM GJ
